tests/regression/tests/REQUEST-942-APPLICATION-ATTACK-SQLI/942160.yaml

```diff
@@ -171,3 +171,19 @@ tests:
         output:
           log:
             expect_ids: [942160]
+  - test_id: 11
```

Comment:
Please also add a negative test, to show why we need `REQUEST_FILENAME`.

Reply:
Not sure if I understood this correctly. I added some negative tests with invalid sleep and benchmark syntax, that do not match. This change does not exactly shoud why we would need `REQUEST_FILENAME` but proves that it does not match broader than before.

Reply:
You're right, I didn't look closely enough.